src/main/java/org/folio/auth/authtokenmodule/apis/RouteApi.java

```diff
@@ -58,8 +60,9 @@ public class RouteApi extends Api implements RouterCreator, TenantInitHooks {
    *                     shared among
    *                     all Api classes.
```

Comment:
Don't forget to update the javadoc with your new param

Reply:
done